packages/neon-core/src/sc/manifest/ContractFeatures.ts

```diff
@@ -0,0 +1,21 @@
+export interface ContractFeaturesLike {
+  storage: boolean;
+  payable: boolean;
+}
+
+export class ContractFeatures {
+  public hasStorage: boolean;
+  public payable: boolean;
+
+  public constructor(hasStorage?: boolean, payable?: boolean) {
```

Comment:
- Overload the constructor to accept an object like all data classes.
- You can accept defaults instead of just optional args.

Reply:
Done